src/sonic-py-common/sonic_py_common/multi_asic_device_info.py

```diff
@@ -0,0 +1,292 @@
+import glob
+import importlib
+import os
+from os import name
```

Comment:
Your import the entire os module on the line above. Remove this and reference as `os.name()` (I don't even see this being used anywhere in the file).

Reply:
Changed in the latest commit